src/events/message/messageCreate.ts

```diff
@@ -24,6 +25,31 @@ export default class extends Event {
 
 	public async run(message: Message): Promise<any> {
 		const context = await getCommandContext(message);
+		Sentry.setContext("user", {
+			tag: context.author.tag,
+			id: context.author.id, // @ts-ignore
+			channelPerms: context.channel.permissionsFor(context.author).bitfield || -1,
+			guildPerms: context.member!.permissions.bitfield,
+		});
+		Sentry.setContext("client", {
+			// @ts-ignore
+			channelPerms: context.channel.permissionsFor(this.client.user).bitfield || -1,
```

Comment:
```suggestion
			channelPerms: context.channel?.permissionsFor(context.author).bitfield || -1,
			guildPerms: context.member!.permissions.bitfield,
		});
		Sentry.setContext("client", {
			// @ts-ignore
			channelPerms: context.channel?.permissionsFor(this.client.user).bitfield || -1,
```
Crash risk if not using optional chaining if a DM event is ever processed.